lib/client.js

```diff
@@ -141,6 +143,47 @@ class Client extends EventEmitter {
   ref() {
     return new Ref(this, ...arguments);
   }
+
+  cast(value, to, alias) {
+    return new Cast(this).cast(value, to, alias);
```

Comment:
if new Cast is only supposed to be called from client, then you can actually pass correct compiler from here as a constructor param

Reply:
I try quickly but it's seems I cant use columnBuilder function because it's depends of args from tableBuilder, type, args. But maybe you want getCompiler function (overrided in each Client) that return the columnCompiler prototype ?